bin/create-release

```diff
@@ -16,22 +16,24 @@ require 'tzinfo'
 REPO = 'code-dot-org/code-dot-org'.freeze
 
 def git_generate_changelog
-  changes, _err, status = Open3.capture3("git log --merges --oneline production^1..production^2 --format=\"%s %b\"")
+  changes, _err, status = Open3.capture3("git log --oneline production^1..production^2 --format=\"%s %b\"")
 
   # command will error out unless the latest commit is a merge
   return nil if status.exitstatus != 0
 
-  re = /Merge pull request #(?<pr>\d*) from code-dot-org\/(?<branch>\S*) (?<description>.*)/
+  # Finds PRs that were either merged or squash-merged
+  merge_commit_pattern = /Merge pull request #(?<pr>\d*) from code-dot-org\/(?<branch>\S*) (?<description>.*)/
+  squash_merge_pattern = /(?<description>.*) \(#(?<pr>\d*)\)/
```

Comment:
can you check if you get any false positives with this regex? it looks a bit more permissive than the one I experimented with earlier. improvements could include looking for at least 5 digits, and requiring a newline after the `)`.

Reply:
Good question - looking for at least 5 digits sounds like a good improvement. I noticed a couple other false positives from this from some older commits, so I'll tinker with this syntax a bit more.

Reply:
any luck Sanchit? I didn't see any false positives in the past 5 years with 5 digit minimum and trailing newline as part of the regex.

Reply:
@davidsbailey thanks for the ping - haven't had a chance to test this in a while, but I think the only false positives I saw were from >5 years ago, so seems pretty safe to go with what you have. I'll confirm and update this.